docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -44,10 +44,10 @@ Change directories to *FSNetCore* and start adding projects to the solution fold
 Use the `dotnet new` command, create a Class Library project in the **src** folder named Library. 
 
 ```bash
-dotnet new classlib -lang F# -o src/Library 
+dotnet new lib -lang F# -o src/Library 
```

Comment:
Yeah, I think that would be a good idea.  Even though most F# developers would understand "Class Library" to mean "Library", especially for new people it might be confusing.